flake.nix

```diff
@@ -1,118 +1,20 @@
 {
-  description = "The Marlowe language";
```

Comment:
@zeme-wana, please review the old nix files to see that I didn't drop any capabilities in migrating to IOGX. I've tested all of the executables, but something else might have been missed.